proto/allocation/allocation.proto

```diff
@@ -55,6 +55,12 @@ message AllocationRequest {
   // Metadata is optional custom metadata that is added to the game server at
   // allocation. You can use this to tell the server necessary session data
   MetaPatch metadata = 7;
+
```

Comment:
also add a deprecation notice to the proto comments. 

Also add `[deprecated = true]` to the old fields as described in https://developers.google.com/protocol-buffers/docs/proto3#options

Reply:
Done.